libultraship/libultraship/GlobalCtx2.cpp

```diff
@@ -1,4 +1,5 @@
 #include "GlobalCtx2.h"
+#include <filesystem>
```

Comment:
Either put this in `GlobalCtx2.h` or above the `#include "GlobalCtx2.h"` directive. The first option is probably the best if you want to use `GlobalCtx2.h` elsewhere.